add know issue part for README.md (#30)
gardener/ops-toolbelt: README.md

```bash
$ .ci/build
```

### Known issues

> 1. Currently there's an know issue for switching to `/bin/sh`, we implemented an color scheme and also added some helper function to display in `/bin/bash` terminal which doesn't work in `/bin/sh`, as workaround when you want to use some script which by defailt needs to utilize `/bin/sh` please use following command to force it utilize `/bin/bash`: (take `chroot` for example)

Reviewer: some minor changes and typo fixes:
```suggestion
1. Currently there's a known issue when using `/bin/sh`. We implemented a color scheme and also added some helper function to display in `/bin/bash` terminal which doesn't work in `/bin/sh`. As workaround when you want to use some script which by default needs to utilize `/bin/sh` please use `/bin/bash` instead if possible: (take `chroot` for example)
```
